src/pybind/mgr/dashboard/frontend/src/app/shared/helpers/helper.ts

```diff
@@ -0,0 +1,2 @@
+export const pluralize = (count: number, value: string): string =>
+  `${value}${count > 1 ? 's' : ''}`;
```

Comment:
we do have a pipe 
https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/pluralize.pipe.ts